### ISSUE TITLE
No connection implementation to wrap GraphQL::Relay::RelationConnection

### ISSUE THREAD

**Issue by mbajur** [rmosolgo/graphql-ruby#432]
Hello once again,

i'm trying to understand why one scenario works while the second one fails (both presented below). I have a lot of (working) connections defined in the exact same way which makes me even more unsure about that.

### Query
```
{
  jobLogs {
    edges {
      node {
        id
      }
    }
  }
}
```

### Working example:
```
QueryType = GraphQL::ObjectType.define do
  connection :jobLogs, JobLogType.connection_type do
    resolve(JobLogsResolver.new)
  end
end
```

### Broken example:
```
QueryType = GraphQL::ObjectType.define do
  connection :jobLogs, JobLogsField
end

JobLogsField = GraphQL::Field.define do
  name 'jobLogs'
  type JobLogType.connection_type

  resolve(JobLogsResolver.new)
end

class JobLogsResolver
  def call(obj, args, ctx)
    JobLog.all
  end
end
```

Running the same query for the second example throws 

    RuntimeError (No connection implementation to wrap GraphQL::Relay::RelationConnection (#<GraphQL::Relay::RelationConnection:0x007f2a0e2e7158>))`

As i've said before, i have like 10 other connections defined in the exact same way and they are working just fine. The only difference between those working ones and this one is a name difference (those are called Job, Video etc. while this one is called JobVideo), maybe graphql don't know hot to resolve it's name or something like this. I honestly have no idea.

Would be really thankfull if anyone could help me with this :) Thanks in advance!

**Comment by rmosolgo** [comment]
This is the same error as #419, here's my take: https://github.com/rmosolgo/graphql-ruby/issues/419#issuecomment-262865466

TL;DR: gem bug, fixed in 1.2.5!

**Comment by rmosolgo** [comment]
Hope that worked for you!

**Comment by mbajur** [comment]
Yes, sorry, i totally forgot to write back. It totally worked for me, thanks a lot! :)
